javascript/javascript1/week3/travelTime.js

```diff
@@ -0,0 +1,19 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 465,
+};
+//console.log(travelInformation["speed"]);
+function travelInfo(travelInformation) {
+  speedMinutes = travelInformation["speed"] / 60;
```

Comment:
Honestly, I prefer[ using `dot notation` over `bracket notation`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_Accessors) to access objects' properties, this will help you to get auto-complete at your editor, and easier of you want to rename the property.  use `travelInformation.speed`